packages/mdc-select/mdc-select.scss

```diff
@@ -89,6 +89,8 @@
 
     // stylelint-disable-next-line selector-max-type, plugin/selector-bem-pattern
     > option {
+      @include mdc-theme-prop(background-color, surface);
```

Comment:
What effect does this have on native selects?

Reply:
This causes the `background-color` to be the `surface` color on browsers that allow it. Since we're changing the color on Line 94, we also need to adjust the background-color. On browsers that don't allow modifications to the dropdown options, neither of these apply and the dropdown is rendered with a white background and black text.